bread/urls.py

```diff
@@ -108,3 +109,7 @@
         browseview=administration.TaskResultBrowseView,
     ),
 ] + external_urlpatterns
+
+for app in apps.get_app_configs():
+    if app.name.startswith("bread.contrib"):
+        urlpatterns.append(path("contrib/", include(f"{app.name}.urls")))
```

Comment:
Saves us from registering all bread contrib urls manually (also wrong branch, should go to main)